packages/w3up-client/src/client.js

```diff
@@ -233,12 +235,50 @@ export class Client extends Base {
 
   /**
    * Create a new space with a given name.
+   * If an account is provided in the options argument, then it creates a delegated recovery account
+   * by provisioning the space and then delegating access to the recovery account.
+   *
+   * @typedef {object} CreateOptions
+   * @property {Account.Account} [account]
    *
    * @param {string} name
+   * @param {CreateOptions} options
+   * @returns {Promise<import("./space.js").OwnedSpace>} The created space owned by the agent.
    */
-  async createSpace(name) {
-    return await this._agent.createSpace(name)
+  async createSpace(name, options = {}) {
+    const space = await this._agent.createSpace(name)
+
+    const account = options.account
+    if (account) {
+      // Provision the account with the space
+      const provisionResult = await account.provision(space.did())
+      if (provisionResult.error) {
+        throw new Error(
+          `⚠️ Failed to provision account: ${provisionResult.error.name}:${provisionResult.error.message}`
+        )
```

Comment:
I recommend against using emojis in error messages - let a UI add that.

```suggestion
        throw new Error(`failed to provision account: ${provisionResult.error.message}`, { cause: provisionResult.error })
```